vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -100,6 +109,11 @@ int main(int argc, char** argv)
         l_rc = l_vpdToolObj.readKeyword(l_vpdPath, l_recordName, l_keywordName,
                                         l_isHardwareOperation, l_filePath);
     }
+    else if (!l_dumpObjFlag->empty())
+    {
+        vpd::VpdTool l_vpdToolObj;
+        l_rc = l_vpdToolObj.dumpObject(l_vpdPath);
```

Comment:
should we check l_vpdPath is empty, incase if Object path is provided like -O "" ?

Reply:
If `-O ""`  is given, the DBus call will fail and `vpd-tool` exits with `rc = -1`.  I do not see any advantage of adding a check for this here.

Reply:
It means we are not validating the input, instead we should catch and say give input is not valid(empty path), instead sending the empty input request to dbus. ?

Reply:
I tried out following two scenarios:

```
root@p10bmc:/tmp# ./vpd-tool -o -O
--object: 1 required TEXT missing
Run with --help for more information.
root@p10bmc:/tmp# ./vpd-tool -o -O ""
Given path is empty.
```
Empty object path is already handled.

Reply:
okay